src/Microsoft.Health.DeID.SharedLib/Exceptions/DeIDFunctionErrorCode.cs

```diff
@@ -0,0 +1,13 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.DeID.SharedLib.Exceptions
+{
+    public enum DeIDFunctionErrorCode
```

Comment:
I think it would be better to use a unified word across the project. Currently some use `deid` while some others use `anonymize`.